logbook-core/src/main/java/org/zalando/logbook/FilteredHttpRequest.java

```diff
@@ -42,11 +45,17 @@ public String getQuery() {
         final String query = request.getQuery();
         return query.isEmpty() ? query : queryFilter.filter(query);
     }
-
+    
     @Override
     public Map<String, List<String>> getHeaders() {
         return headers;
     }
+    
+    @Override
+    public String getPath() {
+        final String path = request.getPath();
```

Comment:
Should this be cached, similar to headers?

Reply:
Why not, also adding caching for query to be consistent then.